rodrigo-espinoza/rodrigo-espinoza-homework-1/src/LinkedListUtils.java

```diff
@@ -0,0 +1,71 @@
+
+
+import java.util.Collections;
+import java.util.LinkedList;
+
+/*
+ * SD2x Homework #1
+ * Implement the methods below according to the specification in the assignment description.
+ * Please be sure not to change the signature of any of the methods!
+ */
+
+public class LinkedListUtils {
+	
+	public static void insertSorted(LinkedList<Integer> list, int value) {
+	    if (list == null) {
+		return;
+	    }else {
+	            if (list.isEmpty() || list.getLast() <= value){
+	                list.add(value);
+	            } else {
+	                for (int i = 0; i < list.size(); i++){
+	                    if (value <= list.get(i)){
+	                        list.add(i, value);
+	                        break;
+	                    }
+	                }
+	            }
+	    }
+	        
+	}
+	
+
+	public static void removeMaximumValues(LinkedList<String> list, int N) {
+	    
+	    if(list == null || N < 0) {		
+		return;
+	    }
+	    if (list.size() <= N) {
+                list.clear();
+                return;
+	    }else {
+		for (int i = 0; i < N; i++) {
+	                LinkedList<String> list2 = new LinkedList<>(list);
+	                list2.sort(Collections.reverseOrder());
+	                list.removeAll(Collections.singletonList(list2.get(0)));
+	            }
+	    }
+	}
+	
+	public static boolean containsSubsequence(LinkedList<Integer> one, LinkedList<Integer> two) {
```

Comment:
20 tests passed.
6 tests failed.
#1. containsSubsequence returns true when first LinkedList contains all elements of second but not in same order
#2. containsSubsequence returns true when lists overlap at end of first LinkedList but it does not contain all elements of second
#3. containsSubsequence returns true when lists have some overlapping elements but first does not contain all elements of second
#4. containsSubsequence returns true when lists have no overlapping elements
#5. containsSubsequence returns true when first LinkedList contains all elements of second but with other elements in sequence
#6. containsSubsequence returns true when first LinkedList contains all elements of second but with other elements in sequence
Your score for this assignment would be 76%

please verify that the first list contain the second in the exact same sequence